runtime/common/src/claims.rs

```diff
@@ -301,38 +429,105 @@ impl<T: Trait> sp_runtime::traits::ValidateUnsigned for Module<T> {
 	fn validate_unsigned(_source: TransactionSource, call: &Self::Call) -> TransactionValidity {
 		const PRIORITY: u64 = 100;
 
-		match call {
+		let maybe_signer = match call {
 			// <weight>
 			// Base Weight: 370 µs
 			// DB Weight: 1 Read (Claims)
 			// </weight>
```

Comment:
I think this comment is misplaced: 
1. the read doesn't happen in this match arm.
2. OTOH, there is a read in the `claim_attest` arm (`Signing`) and one unconditional read below (`Claims`).
3. Probably the comment regarding the timing also got stale.

Consider updating or removing the comment.

Reply:
outside the scope of this PR (the code is unchanged), but i think it will be addressed in @shawntabrizi 's upcoming re-working of the module's weights.

Reply:
yes i will fix